src/js/features/better-viewer-list.js

```diff
@@ -21,12 +22,18 @@ function renderViewerList() {
 
         // Filter empty subsections
         for (var i = 0; i < results.length - 1; i++) {
+            if (!results[i].filter) count += 1;
             if (results[i].filter && results[i + 1].text === ' ') i++;
             else tmpResults.push(results[i]);
         }
         results = tmpResults;
+    } else {
+        count = chatterCount;
     }
     viewList.render(results);
+
+    var label = count + (count < 2 ? ' viewer' : ' viewers');
```

Comment:
1. Your counting system seems pointless. Why not just `results.length`?
2. "chatters" not "viewers"